cocos/2d/CCActionInterval.cpp

```diff
@@ -327,11 +328,10 @@ void Sequence::startWithTarget(Node *target)
         log("Sequence::startWithTarget error: _actions[0] or _actions[1] is nullptr!");
         return;
     }
-    if (_duration <= FLT_EPSILON)
-        return;
+    if (_duration > FLT_EPSILON)
+        _split = _actions[0]->getDuration() / _duration;
```

Comment:
Looks like you merged anyway, but to include an answer here _duration == FLT_EPSILON if it was 0 during initialization.